react/features/toolbox/actions.native.js

```diff
@@ -23,10 +23,13 @@ import {
  */
 export function changeLocalRaiseHand(handRaised: boolean): Function {
     return (dispatch: Dispatch<*>, getState: Function) => {
-        const state = getState();
-        const { secondaryToolbarButtons } = state['features/toolbox'];
+        const { primaryToolbarButtons, secondaryToolbarButtons }
+            = getState()['features/toolbox'];
+
         const buttonName = 'raisehand';
-        const button = secondaryToolbarButtons.get(buttonName);
+        const button
+            = primaryToolbarButtons.get(buttonName)
+            || secondaryToolbarButtons.get(buttonName);
```

Comment:
I see one more places where this can happen, toggleFullScreen. Maybe it's best to just fix it now in the same way to prevent errors.